content/zh/docs/cluster-administration/cluster-settings/cluster-visibility-and-authorization.md

```diff
@@ -1,63 +1,63 @@
 
-   ![select-a-cluster](/images/docs/cluster-administration/cluster-settings/cluster-visibility-and-authorization/select-a-cluster.png)
+   ![选择集群](/images/docs/zh-cn/project-user-guide/cluster-settings/cluster-visibility-and-authorization/select-a-cluster.PNG)
 
-5. After the workspace is created, workspace members with necessary permissions can create resources that run on the associated cluster.
+5. 创建企业空间后，拥有必要权限的企业空间成员可以创建资源，在关联集群上运行。
 
-   ![create-project](/images/docs/cluster-administration/cluster-settings/cluster-visibility-and-authorization/create-project.png)
+   ![创建项目](/images/docs/zh-cn/project-user-guide/cluster-settings/cluster-visibility-and-authorization/create-project.png)
 
    {{< notice warning >}}
 
-Try not to create resources on the host cluster to avoid excessive loads, which can lead to a decrease in the stability across clusters.
+尽量不要在 Host 集群上创建资源，避免负载过高导致多集群稳定性下降。
 
 {{</ notice >}} 
 
-### Set cluster visibility after a workspace is created
+### 在创建企业空间后设置集群可见性
 
-After a workspace is created, you can allocate additional clusters to the workspace through authorization or unbind a cluster from the workspace. Follow the steps below to adjust the visibility of a cluster.
+创建企业空间后，您可以通过授权向该企业空间分配其他集群，或者将集群从企业空间中解绑。按照以下步骤调整集群可见性。
 
-1. Log in KubeSphere with an account that has the permission to manage clusters, such as `admin`.
+1. 使用拥有集群管理权限的帐户登录 KubeSphere，例如 `admin`。
 
-2. Click **Platform** in the top left corner and select **Clusters Management**. Select a cluster from the list to view cluster information.
+2. 点击左上角的**平台管理**，选择**集群管理**。从列表中选择一个集群查看集群信息。
 
-3. In **Cluster Settings** from the navigation bar, select **Cluster Visibility**.
+3. 在左侧导航栏找到**集群设置**，选择**集群可见性**。
 
-4. You can see the list of authorized workspaces, which means the current cluster is available to resources in all these workspaces.
+4. 您可以看到已授权企业空间的列表，这意味着所有这些企业空间中的资源都能使用当前集群。
 
-   ![cluster-visibility-settings-1](/images/docs/cluster-administration/cluster-settings/cluster-visibility-and-authorization/cluster-visibility-settings-1.png)
+   ![设置集群可见性1](/images/docs/zh-cn/project-user-guide/cluster-settings/cluster-visibility-and-authorization/workspace-list.PNG)
 
-5. Click **Edit Visibility** to set the cluster authorization. You can select new workspaces that will be able to use the cluster or unbind it from a workspace.
+5. 点击**编辑可见范围**设置集群授权。您可以选择让新的企业空间使用该集群，或者将该集群从企业空间解绑。
 
-   ![cluster-visibility-settings-2](/images/docs/cluster-administration/cluster-settings/cluster-visibility-and-authorization/cluster-visibility-settings-2.png)
+   ![设置集群可见性2](/images/docs/zh-cn/project-user-guide/cluster-settings/cluster-visibility-and-authorization/assign-workspace.PNG)
 
-### Make a cluster public
+### 将集群设置为公开集群
 
-You can check **Set as public cluster** so that all platform users can access the cluster, in which they are able to create and schedule resources.
+您可以打开**设置为公开集群**，这样所有平台用户都可以访问该集群，在该集群上创建并调度资源。
```

Comment:
这样所有平台用户都可以访问该集群，在该集群上创建并调度资源。
->
以便平台用户访问该集群，并在该集群上创建和调度资源。

The original English text is not accurate enough. Change the English source text to:

You can check **Set as public cluster** so that platform users can access the cluster, in which they are able to create and schedule resources.